pkg/mimir/mimir_test.go

```diff
@@ -427,6 +433,91 @@ func TestFlagDefaults(t *testing.T) {
 	require.Equal(t, 10*time.Second, c.Server.GRPCServerMinTimeBetweenPings)
 }
 
+// TODO Remove in Mimir 2.2.
+//      Previously ActiveSeriesCustomTrackers was an ingester config, now it's in LimitsConfig.
+//      We provide backwards compatibility for it by parsing the old YAML location and copying it to LimitsConfig here,
+//      unless it's also defined in the limits, which is invalid.
+//		This needs to be set before setting default limits for unmarshalling.
+// 		For more context see https://github.com/grafana/mimir/pull/1188#discussion_r830129443
+func (t *Mimir) initTest() (services.Service, error) {
+
+	return services.NewTimerService(time.Second,
```

Comment:
We can use `NewBasicService` to make test faster (now it waits 1 second before running the function). Test logic should be in `running` function.